prow/plugins/projectmanager/projectmanager_test.go

```diff
@@ -0,0 +1,678 @@
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			pe: github.PullRequestEvent{
+				Action: github.PullRequestActionOpened,
+				Number: 1,
+				PullRequest: github.PullRequest{
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg2",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{
+				4: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+			},
+		},
+		{
+			name: "add pull request using column name in multirepo to multiple projects",
+			gc: &fakegithub.FakeClient{
+				IssueLabelsAdded:   labels,
+				IssueLabelsRemoved: []string{},
+				RepoProjects: map[string][]github.Project{
+					"testOrg/testRepo": {
+						{
+							Name: "testProject",
+							ID:   1,
+						},
+					},
+					"testOrg/testRepo2": {
+						{
+							Name: "testProject2",
+							ID:   2,
+						},
+					},
+				},
+				OrgProjects: map[string][]github.Project{},
+				ProjectColumnsMap: map[string][]github.ProjectColumn{
+					"testProject": {
+						{
+							Name: "testColumn2",
+							ID:   2,
+						},
+						{
+							Name: "testColumn",
+							ID:   1,
+						},
+					},
+					"testProject2": {
+						{
+							Name: "testColumn2",
+							ID:   4,
+						},
+						{
+							Name: "testColumn",
+							ID:   3,
+						},
+					},
+				},
+				ColumnCardsMap: map[int][]github.ProjectCard{},
+				ColumnIDMap:    columnIDMap,
+			},
+			projectManager: plugins.ProjectManager{
+				OrgRepos: map[string]plugins.ManagedOrgRepo{
+					"testOrg/testRepo": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject": {
+								Columns: []plugins.ManagedColumn{
+									{
+										Name:   "testColumn",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1"},
+									},
+								},
+							},
+						},
+					},
+					"testOrg/testRepo2": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject2": {
+								Columns: []plugins.ManagedColumn{
+									{
+										Name:   "testColumn2",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1", "label2"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			pe: github.PullRequestEvent{
+				Action: github.PullRequestActionOpened,
+				Number: 1,
+				PullRequest: github.PullRequest{
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{
+				4: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+				1: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+			},
+		},
+		{
+			name: "add pull request using column name to multiple columns in a project should fail",
```

Comment:
Why is there no `expectedError` field specified for this test case? This makes me worry that the tests are not actually validating the plugin properly.

Reply:
The fix currently is to not throw errors if we fail to add a project card, instead we log the issue why we failed to add and proceed.